tools/packaging/kernel/build-kernel.sh

```diff
@@ -66,6 +66,8 @@ kernel_url=""
 linux_headers=""
 # Enable measurement of the guest rootfs at boot.
 measured_rootfs="false"
+# Enable dm-verity in the guest.
+dm_support="false"
```

Comment:
This is confusing since DM is not necessarily `dm-verity`. Maybe rename to `dm_verity_support=`?

Reply:
Fixed.